Assets/Scripts/AI/RandomAI.cs

```diff
@@ -0,0 +1,20 @@
+﻿using UnityEngine;
+using Assets.Scripts.Board;
+using System.Collections.Generic;
+
+public class RandomAI : AIBehavior
+{
+
+    public override void Start()
+    {
+        base.Start();
+    }
+
+    override protected GameCell GetNextMove()
+    {
+        Vector2 currentPosition = positionConverter.ConvertScenePositionToBoard(this.transform.position);
+        List<GameCell> adjacentCells = board.GetAdjacentCells(currentPosition);
+        GameCell nextCell = adjacentCells[Random.Range(0, adjacentCells.Count)];
+        return nextCell;
```

Comment:
Pytanie jak chcemy to rozwiązać. Możemy np. pozwolić potworkom wchodzić na pola zajęte przez gracza (wtedy mu odejmują życie), a zablokować tylko na pola zajęte przez bomby i cube'y.
Druga możliwość to niech stoi w miejscu i robić pewnego rodzaju pooling w Update(), który będzie cały czas sprawdzał, czy się nie zwolniło jakieś pole obok i jak będzie wolne to z powrotem wrzuci go w pętlę, która go porusza. @dymara?